aws_lambda_powertools/event_handler/openapi/constants.py

```diff
@@ -1,2 +1,2 @@
 DEFAULT_API_VERSION = "1.0.0"
-DEFAULT_OPENAPI_VERSION = "3.0.3"
+DEFAULT_OPENAPI_VERSION = "3.1.0"
```

Comment:
do we already have the Bedrock Agent workaround as their docs are wrong?

Reply:
We will resolve issues with Agent Bedrock in this other PR @heitorlessa

https://github.com/aws-powertools/powertools-lambda-python/issues/4196